pkg/diagnostics/http_monitoring.go

```diff
@@ -162,15 +191,26 @@ func (h *httpMetrics) ClientRequestStarted(ctx context.Context, method, path str
 		return
 	}
 
+	normalizedPath, ok := h.normalizePath(path, Ingress)
+	if ok {
+		path = normalizedPath
+	}
+
 	if h.legacy {
 		stats.RecordWithTags(
 			ctx,
 			diagUtils.WithTags(h.clientSentBytes.Name(), appIDKey, h.appID, httpPathKey, h.convertPathToMetricLabel(path), httpMethodKey, method),
 			h.clientSentBytes.M(contentSize))
 	} else {
+		tags := diagUtils.WithTags(h.clientSentBytes.Name(), appIDKey, h.appID)
+
+		if h.pathNormalization.enabled {
+			tags = append(tags, diagUtils.WithTags(h.clientSentBytes.Name(), httpPathKey, path, httpMethodKey, method)...)
```

Comment:
Here too, avoid using `append` and just have 2 else's:

- `else if h.pathNormalization.enabled` and `else`